src/views/Tasks/index.vue

```diff
@@ -1,14 +1,239 @@
+      <span class="red--text">"body"</span> : <span class="red--text">"The body of the article"</span>
+      <span class="red--text">"author"</span> : <span class="red--text">"The author"</span>
+    },
+    ...
+  ]
+          </pre>
+          </v-card-text>
+        </v-card>
+        <br />
+        </div>
+  
+        <div id="get-by-id">
+     <h3 class=title>Get a specific article by ID</h3>
+        <v-divider></v-divider>
+        <br />
+        <p>A GET request on route <code>"/articles/ID"</code> should return the article with the respective ID.
+        If the article does not exist, then the request should return error 404.
+        </p>
+  
+        <v-card >
+          <v-card-subtitle>
+            GET /articles/ID response:
+          </v-card-subtitle>
+          <v-card-text>
+            <pre>          
+  {
+    <span class="red--text">"_id"</span> : <span class="red--text">"ID"</span>
+    <span class="red--text">"title"</span> : <span class="red--text">"The Title"</span>
+    <span class="red--text">"subtitle"</span> : <span class="red--text">"The subtitle"</span>
+    <span class="red--text">"body"</span> : <span class="red--text">"The body of the article"</span>
+    <span class="red--text">"author"</span> : <span class="red--text">"The author"</span>
+  }
+          </pre>
+          </v-card-text>
+        </v-card>
+        <br />
+        </div>
+  
+        <div id="update-article">
+        <h3 class=title>Update article</h3>
+        <v-divider></v-divider>
+        <br />
+        <p>A PUT request on the <code>"/articles/ID" </code> route shall replace the entry corresponding to the article's id in the database.
+           If the given article is invalid (i.e. the body, title, subtitle or lead paragraph is either empty), the route shall return a 400.
+           If an article with the same id does not exist in the database, the route shall return a 404.
+           If the request is a success, the route shall return { "message": "Success" }.
+           For example, to change the title of an article with id 'ID', the PUT request would look like this: 
+  
+        </p>
+  
+        <v-card >
+          <v-card-subtitle>
+            PUT /articles/ARTICLE_ID request:
+          </v-card-subtitle>
+          <v-card-text>
+            <pre>          
+  {
+    <span class="red--text">"title"</span> : <span class="red--text">"New Title"</span>
+  }
+          </pre>
+          </v-card-text>
+        </v-card>
+        <br />
+        </div>
+  
+        <div id="delete-article">
+        <h3 class=title>Delete article</h3>
+        <v-divider></v-divider>
+        <br />
+        <p>A DELETE request on route <code>/articles/ARTICLE_ID</code> should remove the article with id 'ARTICLE_ID' from the database.
+        If no article exists with the given id, then request should return error status 404.
+        If the request was bad (e.g invalid id, ), the the request should return error message 400.
```

Comment:
1. unnecessary comma and space after id
2. should respond with a status code 400
3. if successful, respond with status code 200